module3/homework/advanced_calculator/advancedCalculator.cpp

```diff
@@ -0,0 +1,164 @@
+#include <algorithm>
+#include <cmath>
+#include <functional>
+#include <map>
+#include <string>
+
+#include "advancedCalculator.hpp"
+
+const std::string operations = "+-*/%$^!";
+const std::string digits = "1234567890.";
+
+const std::map<std::string, std::function<double(double, double)>> command = {{"+", std::plus<double>()},
```

Comment:
You can move elements to the new line, now its hard to read

Reply:
Good point. In was in a big hurry so there might be more problems like this 😔